### PR TITLE
Offer only supported sector sizes in NVDIMM reconfiguration dialog

### PR DESCRIPTION
**Pull request by rvykydal**
Resolves: rhbz#1614049

### CODE REVIEW

File: pyanaconda/ui/gui/spokes/advstorage/nvdimm.py [hunk 1]
```diff
@@ -38,6 +38,9 @@
 PAGE_RESULT_ERROR = 2
 PAGE_RESULT_SUCCESS = 3
 
+NVDIMM_SUPPORTED_SECTOR_SIZES = ['512', '4096']
```
Comment (jkonecny12) [inline pyanaconda/ui/gui/spokes/advstorage/nvdimm.py]: Aren't we able to get these values from blivet?

Comment (rvykydal) [reply pyanaconda/ui/gui/spokes/advstorage/nvdimm.py]: We are not.
@vojtechtrefny - would it make sense to provide such values ?

Comment (rvykydal) [reply pyanaconda/ui/gui/spokes/advstorage/nvdimm.py]: After talking to Vojta I am just going to rename the constant.
There seems to be about 8 sizes supported for sector mode currently, but we don't want to offer all of them in UI. Vojta will probably propagate info about supported sizes by libblockdev api, and we may use it to check kickstart values, but it should be solved in a separate issue.

Comment (jkonecny12) [reply pyanaconda/ui/gui/spokes/advstorage/nvdimm.py]: Nice cooperation thanks!

**Comment by rvykydal** [comment]
The test have actually passed.

**Comment by rvykydal** [comment]
Ported to master by https://github.com/rhinstaller/anaconda/pull/2323

**Pull request merged**